paddle/fluid/framework/framework.proto

```diff
@@ -126,6 +126,7 @@ message VarType {
     LOD_TENSOR_ARRAY = 13;
     PLACE_LIST = 14;
     READER = 15;
+    FETCH_VAR_LIST = 16;
```

Comment:
I feel FETCH_VAR_LIST is very similar to FETCH_LIST by meaning. Could you find whether we can reuse the FETCH_LIST type?